ENIGMAsystem/SHELL/Universal_System/Extensions/FileDropper/FileDropper.cpp

```diff
@@ -91,6 +89,9 @@ LRESULT CALLBACK SetHook(int nCode, WPARAM wParam, LPARAM lParam) {
 }
 
 HHOOK InstallHook() {
+  ChangeWindowMessageFilterEx(enigma::hWnd, WM_DROPFILES, MSGFLT_ADD, NULL);
+  ChangeWindowMessageFilterEx(enigma::hWnd, WM_COPYDATA, MSGFLT_ADD, NULL);
+  ChangeWindowMessageFilterEx(enigma::hWnd, 0x0049 /* = WM_COPYGLOBALDATA */, MSGFLT_ADD, NULL);
```

Comment:
Maybe just `#define` it? I think this one is in `user.h` or something.

Reply:
I didn't want to define it because we might switch to the MS compiler eventually and there would be a naming conflict with an existing macro in that case. This way ensures it will work with both compilers while giving away what it is in a comment. Let me know if you think it still needs to be #define'd regardless.

Reply:
@RobertBColton apparently Travis and Appveyor run Vista or older because both CI's failed when I used ChangeWindowMessageFilterEx which is Windows 7 onwards and defined in <windows.h>, and the CI errors said the function was an undefined reference. We'll have to scrap that idea until we can upgrade our CI to use Windows 7 or newer only.

Reply:
That's fine, we support Vista or newer (and XP where we still can). But `#ifndef` solves your problem above and allows you to define it only if it's not already defined like GLFW does.
https://github.com/glfw/glfw/blob/44b5d06583cd21ac237eb8f6263db03faf1726c2/src/win32_platform.h#L73